### PR TITLE
✨proctor exam xblock

### PR DESCRIPTION
**Pull request by rmoch**
edX XBlock to provide exam proctoring using Proctor Exam service
This xblock will log student in PE service by LTI therefore it relies on FUN's [configurable-lti-provider](https://github.com/openfun/xblock-configurable-lti-consumer)

### CODE REVIEW

**Review by sampaccoud — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+;;
+;; FUN package
+;;
+[metadata]
+name = xblock-proctor-exam
+version = 0.3.0
```
Comment (jmaupetit) [inline setup.cfg]: Sure about this release number?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+description =
```
Comment (jmaupetit) [inline setup.cfg:7]: Maybe add the same baseline as the github project?

**Review by jmaupetit — COMMENTED** [review]
Can we make strings translatable in xblock templates?

**Comment by rmoch** [comment]
>Can we make strings translatable in xblock templates?

Yes, why only in templates ?

**Comment by jmaupetit** [comment]
I've seen french in templates, but yes: everything should be translatable.

**Review by jmaupetit — APPROVED** [review]
Maybe declare an issue to make strings translatable later.

**Pull request merged**